internal/config/client.go

```diff
@@ -335,11 +335,6 @@ func (c *Config) UserConfig() ClientConfig {
 		Where("deleted_at IS NULL").
 		Take(&result.Count)
 
-	c.Db().
-		Table("countries").
-		Select("(COUNT(*) - 1) AS countries").
-		Take(&result.Count)
```

Comment:
Should not be removed / replaced as the country album count may be less. Moments will only be created if enough good photos exist for a certain filter, in this case a country code.